docs/index.md

```diff
@@ -1 +1,56 @@
 # JFrog Xray Provider
+
+The [Xray](https://jfrog.com/xray/) provider is used to interact with the
+resources supported by JFrog Xray. Xray is a part of JFrog Artifactory and can't be used separately.
+The provider needs to be configured with the proper credentials before it can be used.
+
+Xray API documentation can be found [here](https://www.jfrog.com/confluence/display/JFROG/Xray+REST+API)
+
+This provider requires access to Artifactory and Xray APIs, which are only available in the _licensed_ pro and enterprise editions.
+You can determine which license you have by accessing the following URL
+`${host}/artifactory/api/system/licenses/`
+
+You can either access it via api, or web browser - it does require admin level credentials, but it's one of the few
+APIs that will work without a license (side node: you can also install your license here with a `POST`)
+
+```bash
+curl -sL ${host}/artifactory/api/system/licenses/ | jq .
```

Comment:
talk to alex - he actually put an extra test check in place re: this

Reply:
https://github.com/jfrog/terraform-provider-artifactory/pull/229

Reply:
should I remove the manual way from the readme then? I have that test as well in provider.go